### PR TITLE
chore(deps): un-patch tokio

### PR DESCRIPTION
**Pull request by hawkw**
now that 1.6.2 has been published we can remove the patch.

Signed-off-by: Eliza Weisman <eliza@buoyant.io>

### CODE REVIEW

**Comment by seanmonstar** [comment]
I don't think the waker instrumentation made it into 1.6.2, it only contained a bug fix. But 1.7.0 does!

File: console-subscriber/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ edition = "2018"
 
 [dependencies]
 
-tokio = { version = "^1.5", features = ["sync", "time", "macros", "tracing"]}
+tokio = { version = "^1.6.2", features = ["sync", "time", "macros", "tracing"]}
```
Comment (hawkw) [inline console-subscriber/Cargo.toml]:
```suggestion
tokio = { version = "^1.7", features = ["sync", "time", "macros", "tracing"]}
```

File: console-subscriber/Cargo.toml [hunk 2]
```diff
@@ -19,7 +19,7 @@ futures = { version = "0.3", default-features = false }
 
 [dev-dependencies]
 
-tokio = { version = "^1.5", features = ["full", "rt-multi-thread"]}
+tokio = { version = "^1.6.2", features = ["full", "rt-multi-thread"]}
```
Comment (hawkw) [inline console-subscriber/Cargo.toml]:
```suggestion
tokio = { version = "^1.7", features = ["full", "rt-multi-thread"]}
```

**Comment by hawkw** [comment]
> I don't think the waker instrumentation made it into 1.6.2, it only contained a bug fix. But 1.7.0 does!

whoops, fixed it!

**Review by seanmonstar — APPROVED** [review]

**Pull request merged**